openedx_lti_tool_plugin/resource_link_launch/views.py

```diff
@@ -249,16 +249,16 @@ def check_course_access_permission(self, course_id: str, iss: str, aud: str):
             return
 
         lti_tool_config = self.tool_config.get_lti_tool(iss, aud)
```

Comment:
```suggestion
        lti_tool = self.tool_config.get_lti_tool(iss, aud)
        lti_tool_configuration = LtiToolConfiguration.objects.filter(
```
@Serafin-dev What do you think about renaming these variables?, I think it's a bit confusing referring both as configurations